internal/util/get/get_test.go

```diff
@@ -320,28 +283,22 @@ func TestCommand_Run_tag(t *testing.T) {
 	g.AssertEqual(t, filepath.Join(g.DatasetDirectory, testutil.Dataset2), r)
 
 	// verify the KptFile contains the expected values
-	g.AssertKptfile(t, r, kptfile.KptFile{
-		ResourceMeta: yaml.ResourceMeta{
-			ObjectMeta: yaml.ObjectMeta{
-				NameMeta: yaml.NameMeta{
-					Name: g.RepoName,
-				},
-			},
-			TypeMeta: yaml.TypeMeta{
-				APIVersion: kptfile.TypeMeta.APIVersion,
-				Kind:       kptfile.TypeMeta.Kind},
-		},
-		PackageMeta: kptfile.PackageMeta{},
-		Upstream: kptfile.Upstream{
-			Type: "git",
-			Git: kptfile.Git{
-				Directory: "/",
-				Repo:      g.RepoDirectory,
-				Ref:       "refs/tags/v2",
-				Commit:    commit,
-			},
-		},
-	})
+
+	contents, err := ioutil.ReadFile(filepath.Join(r, kptfile.KptFileName))
+	assert.NoError(t, err)
+	assert.Equal(t, fmt.Sprintf(`apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
```

Comment:
It seems like the template looks very similar in each of the tests. Can we either avoid the templating all together, or create one template that can be reused by all (or at least most) of the tests?